### ISSUE TITLE
Has this been tested on recent (supported) rpi releases

### ISSUE THREAD

**Issue by davidywilson** [QDeltaSoft/qradiolink#3]
Just wondering if this has been tested on recent versions of rpi OS. 
I tried to install on Raspbian Buster but got errors such as 

"This file was generated by an older version of protoc" etc

There is not a lot of information as to what versions of the various libraries this is compatible with. 
I suspect the latest git hub update was back when wheezy was latest - but that is no longer supported and repo's are no longer valid etc. 

It would be awesome to get this going on buster.

**Comment by kantooon** [comment]
No. All development stopped a long time ago. See the new repo: https://github.com/qradiolink/qradiolink
